ca/django_ca/managers.py

```diff
@@ -336,9 +342,18 @@ def init(
         # NOTE: Already verified by KeySizeAction, so these checks are only for when the Python API is used
         #       directly.
         generate_key_args = validate_key_parameters(key_size, key_type, ecc_curve)
-        algorithm = parse_hash_algorithm(algorithm)
+        algorithm = None if openssh_ca else parse_hash_algorithm(algorithm)
```

Comment:
Please do not write this in a single line, as coverage tests will show this as covered even if (for example) `openssh_ca` is never `True`.

Reply:
Converted to if statement